api/tests/Api/Profiles/UpdateProfileTest.php

```diff
@@ -204,6 +204,53 @@ public function testPatchProfileTrimsLanguage() {
         ]);
     }
 
+    public function testPatchProfileValidatesInvalidColor() {
```

Comment:
is it also allowed to send these properties on create?

Reply:
Yes. But the CreateProfileTest was empty, so I skipped adding them there.

Reply:
Profiles are created with the user together as embedded entity there.
But we will not add it there in the form, and if it is allowed to that with the api, thats kind of ok.